scripts/test.sh

```diff
@@ -2,23 +2,36 @@
 
 set -e
 
-echo "\n\Building packages..."
+echo "\n\nBuild projects..."
 pnpm run build
 
 echo "\n\nStarting docker..."
 pnpm run node:up
 
+RUN_TESTS_TRY=0
+MAX_TRIES=50
```

Comment:
This is not the attempt of running tests, but the time it waits for the nodes to be available. As the integration test needs to wait for the L1 node to be running with all contracts deployed and fuel-core start listening to it.

This means;
- It starts the nodes wait for them to be syncing and with the contracts deployed
- Once both are working then run the tests;
- If the nodes didn't start in 5 minutes it will consider a fail and exit.

Reply:
@Braqzen I think, I can add some better descriptions and improve the variable names here to be clear.